File: app/src/main/java/com/irontec/axier/rxweather/model/Forecast.java
package com.irontec.axier.rxweather.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by axier on 14/7/17.
 */

public class Forecast {

    @SerializedName("forecastday")
    private List<Forecastday> forecastdays;

    public Forecast() {
    }

    public List<Forecastday> getForecastdays() {
        return forecastdays;
    }

    public void setForecastdays(List<Forecastday> forecastdays) {
        this.forecastdays = forecastdays;
    }
}

File: app/src/main/java/com/irontec/axier/rxweather/model/Forecast.kt
package com.irontec.axier.rxweather.model

import com.google.gson.annotations.SerializedName

/**
 * Created by axier on 14/7/17.
 */

class Forecast {

    @SerializedName("forecastday")
    var forecastdays: List<Forecastday>? = null
}
